rpcserver.go

```diff
@@ -5781,14 +5781,23 @@ func (r *rpcServer) GetTransactions(ctx context.Context,
 	// special case end height which will return transactions from the start
 	// height until the chain tip, including unconfirmed transactions.
 	var endHeight = btcwallet.UnconfirmedHeight
-
+	txHashes := make([]*chainhash.Hash, len(req.TxHash))
+	var err error
 	// If the user has provided an end height, we overwrite our default.
 	if req.EndHeight != 0 {
 		endHeight = req.EndHeight
 	}
-
+	if len(req.TxHash) > 0 {
+		for i := range req.TxHash {
+			strHash := hex.EncodeToString(req.TxHash[i])
+			txHashes[i], err = chainhash.NewHashFromStr(strHash)
```

Comment:
Combined with my previous comment, this would now only be `txHashes[i], err = chainhash.NewHash(req.TxHash[i])`.